examples/rcn.py

```diff
@@ -271,18 +272,20 @@ def valid_configs(r: int, hps: int, vps: int) -> np.ndarray:
 
 # %%
 start = time.time()
-fg = graph.FactorGraph(variables=variables_all_models)
+fg = graph.FactorGraph(variables_all_models)
 
 # Adding rcn model edges to the pgmax factor graph.
 for idx in range(edges.shape[0]):
     edge = edges[idx]
 
     for e in edge:
         i1, i2, r = e
-        fg.add_factor(
-            [(idx, i1), (idx, i2)],
-            valid_configs_list[r],
+        factor = EnumerationFactor(
+            variables=[variables_all_models[idx][i1], variables_all_models[idx][i2]],
+            factor_configs=valid_configs_list[r],
+            log_potentials=np.zeros(valid_configs_list[r].shape[0]),
```

Comment:
This should automatically default to an all 0 array if omitted

Reply:
This is true for `EnumerationFactorGroup`, not for `EnumerationFactor`